src/bp-members/classes/class-bp-members-signups-feature.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * BuddyPress Member's notice feature Class.
+ *
+ * @package BuddyPress
+ * @subpackage Members
+ * @since 15.0.0
+ */
+
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * The BP Signups feature is now optional.
+ *
+ * If you want to disable it, you can use:
+ * `add_filter( 'bp_is_members_signups_active', '__return_false' );`
```

Comment:
Curous: is this hook a replacement for the `bp_get_signup_allowed` hook and helper function?

Reply:
No, it's not. This hook is to completely disable the BuddyPress way of dealing with signups. It doesn't mean `bp_get_signup_allowed()` would be false. `bp_get_signup_allowed()` can be `true` but dealing with registration is managed by WordPress or a 3rd party plugin.